### ISSUE TITLE
ops: doc: set up individual pages for each kernel and start filling the basics

### ISSUE THREAD

**Issue by fnrizzi** [Pressio/pressio#413]
IMPORTANT: branch off `develop` , one PR per kernel

The code to look at is here: https://github.com/Pressio/pressio/tree/develop/include/pressio/ops

The documentation source is here: https://github.com/Pressio/pressio/tree/develop/docs/source

Scope: for each kernel implemented in the ops, we need to make individual pages similar to how [fill is done](https://github.com/Pressio/pressio/tree/develop/docs/source/components/ops/fill.rst): 

<img width="1399" alt="image" src="https://user-images.githubusercontent.com/18708772/190965542-2e28810d-a658-40bc-96d6-1c5e4a503f08.png">

For the time being, for each kernel I would like to have at least parameters and constraints spelled out, leaving the other descriptions empty. I like to have pages for all ops finished this way even if we have to fill things later. 
So basically we want to first have all ops out even if partially filled like this:

<img width="870" alt="image" src="https://user-images.githubusercontent.com/18708772/190965168-c902698f-bb50-4d10-98c7-462de7ce0090.png">
